skimage/measure/_regionprops.py

```diff
@@ -383,7 +379,9 @@ def regionprops(label_image, intensity_image=None, cache=True):
     **centroid** : array
         Centroid coordinate tuple ``(row, col)``.
     **convex_area** : int
-        Number of pixels of convex hull image.
+        Number of pixels of convex hull image, which is the smallest convex 
+        polygon that encloses the region. The size of the convex hull image is 
+        described by the bounding box.
```

Comment:
Is not clear if the attribute store the number of pixels of convex polygon or of bounding box.